estimagic/tests/differentiation/test_gradient.py

```diff
@@ -0,0 +1,101 @@
+import os
+import pickle
+
+import pytest
+from pandas.testing import assert_series_equal
+
+from estimagic.differentiation.gradient import gradient
+from estimagic.examples.logit import logit_loglike
+
+
+@pytest.fixture()
+def statsmodels_fixtures():
+    with open(
+        os.path.join(os.path.dirname(__file__), "diff_fixtures.pickle"), "rb"
+    ) as p:
+        fix = pickle.load(p)
+    return fix
+
+
+def test_gradient_forward(statsmodels_fixtures):
+    fix = statsmodels_fixtures
+    assert_series_equal(
+        gradient(
+            logit_loglike,
+            fix["params"],
+            method="forward",
+            extrapolation="",
```

Comment:
This is exactly the reason, why a bool is better than a string here.

Reply:
Adjusted tests.